src/main/java/frc/robot/commands/Rotate.java

```diff
@@ -54,7 +54,7 @@ public void execute() {
 
         m_lastError = m_error;
 
-        m_drivetrain.percentDrive(-percentOutput, percentOutput);
+        m_drivetrain.velocityDrive(-percentOutput, percentOutput);
```

Comment:
rotate relies on percent drive being used, also percent drive was deleted

Reply:
Yup, revert this change and add percent drive back

Reply:
Added percent drive back in drivetrain and changed to percentdrive in rotate.java